adr/000x-developing-and-publishing-clientside-behaviours.md

```diff
@@ -0,0 +1,46 @@
+# 000X. Developing and Publishing ClientSide Behaviours
```

Comment:
What do you think about numbering the ADR's based off the PR number (250, in this case). That way the values are monotonically increasing and we should avoid merge conflicts with concurrent PR's.

Reply:
I'm open to the idea. What do others think? @manuelpuyol @joelhawksley?

Reply:
I'm :+1: with that